3style-text/styles.css

```diff
@@ -0,0 +1,89 @@
+/* ---font styles--- */
+
+body {
+  font-family: "Public Sans", Verdana, sans-serif;
+  font-size: 18px;
+}
+
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: "Quicksand", sans-serif;
+  font-weight: 600;
+}
+
+
+/* ---font sizes--- */
+
+h1 {
+	font-size: 250%; /* 45px */
+}
+
+h2 {
+	font-size: 200%; /* 36px */
+}
+
+h3 {
+	font-size: 177.78%; /* 32px */
+}
+
+h4 {
+	font-size: 162.5%; /* 26px */
+}
+
+h5 {
+	font-size: 122%; /* 22px */
+}
+
+h6 {
+	font-size: 100%; /* 18px */
+}
+
+h3 {
+	font-weight: 700;
+}
+
+/* ---font weights--- */
+
+strong em,
+em strong {
+	font-family: "Quicksand", sans-serif;
+    font-style: normal;
+}
+
+/* ---font colors--- */
+
+body {
+	font-family: "Public Sans", Verdana, sans-serif;
+	font-size: 18px;
```

Comment:
You already have these two lines in the first instance of `body` (line 3). No need to declare twice. I would remove it from here since it doesn't relate to color.

Reply:
I think this is an old file from the style-text branch. What was happening is I was bringing changes from other branches because they were un-staged. I learned my lesson! There should be fewer of these mistakes in the next tutorials :)